ConwaysGameOfLife/Validator.cs

```diff
@@ -5,14 +5,45 @@ namespace ConwaysGameOfLife
     public class Validator
     {
         // validates input
-        public bool IsUniverse(Universe input)
+
+        public static bool ParseInput(string numberInput)
+        {
+            if (Int32.TryParse(numberInput, out var number))
+            {
+                return true;
+            }
+            return false;
+        }
+        
+        public bool IsValidUniverse(string input)
+        {
+            var isNumber = SplitInput(input, ",");
+            if (!input.Contains(",")) return false;
+            if (input.Length > 3) return false;
+            if (isNumber.Length != 2) return false;
+            if (!ParseInput(isNumber[0]) || !ParseInput(isNumber[1])) return false;
+            if (Int32.Parse(isNumber[0]) <= 0) return false;
+            return true;

```

Comment:
Probably worth thinking about the ordering of these e.g. the check for contains "," probably could go first. It could also be a count of the `char` `',' == 1`

The `isNumber` variable name could be something like `inputComponents` or something to do with the input but split.

Reply:
is `input.Length` a determining factor for validity? conceivably the universe could be 10x5, for example, in which case `"10,3"` should be valid. could add this as a test case